src/Templates/src/templates/maui-mobile/Resources/Styles.xaml

```diff
@@ -0,0 +1,164 @@
+        <!--<Setter Property="CharacterSpacing" Value="0" />-->
+        <!--<Setter Property="FontFamily" Value="OpenSansRegular" />-->
+        <!--<Setter Property="FontSize" Value="14" />-->
+        <!--<Setter Property="TextTransform" Value="None" />-->
+        <!--<Setter Property="FontAttributes" Value="Bold" />-->
+        <!--<Setter Property="FontAutoScalingEnabled" Value="True" />-->
+        <!--<Setter Property="BorderWidth" Value="0" />-->
+        <Setter Property="BorderColor" Value="{AppThemeBinding Light={StaticResource AccentColor}, Dark={StaticResource PrimaryLightColor}}" />
+        <!--<Setter Property="CornerRadius" Value="0" />-->
+        <!--<Setter Property="Padding" Value="5" />-->
+        <!--<Setter Property="LineBreakMode" Value="WordWrap" />-->
+
+        <!-- VIEW INHERITED PROPERTIES -->
+        <!--<Setter Property="VerticalOptions" Value="Center" />-->
+        <!--<Setter Property="HorizontalOptions" Value="Center" />-->
+        <!--<Setter Property="Margin" Value="0" />-->
+
+        <!-- VISUALELEMENT INHERITED PROPERTIES -->
+        <!--<Setter Property="Shadow" Value="" />-->
+        <!--<Setter Property="Width" Value="100" />-->
+        <!--<Setter Property="Height" Value="25" />-->
+        <!--<Setter Property="Clip" Value="" />-->
+        <!--<Setter Property="IsVisible" Value="True" />-->
+        <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource AccentColor}, Dark={StaticResource PrimaryLightColor}}" />
+        <!--<Setter Property="Background" Value="" />-->
+        <!--<Setter Property="WidthRequest" Value="100" />-->
+        <!--<Setter Property="HeightRequest" Value="25" />-->
+        <!--<Setter Property="FlowDirection" Value="LeftToRight" />-->
+    </Style>
+
+    <Style TargetType="Grid">
+
+        <!-- GRID SPECIFIC PROPERTIES -->
+        <!--<Setter Property="RowSpacing" Value="5" />-->
+        <!--<Setter Property="ColumnSpacing" Value="5" />-->
+
+        <!-- LAYOUT INHERITED PROPERTIES -->
+        <!--<Setter Property="IsClippedToBounds" Value="False" />-->
+        <!--<Setter Property="Padding" Value="5" />-->
+
+        <!-- VIEW INHERITED PROPERTIES -->
+        <!--<Setter Property="VerticalOptions" Value="Center" />-->
+        <!--<Setter Property="HorizontalOptions" Value="Center" />-->
+        <!--<Setter Property="Margin" Value="0" />-->
+
+        <!-- VISUALELEMENT INHERITED PROPERTIES -->
+        <!--<Setter Property="Shadow" Value="" />-->
+        <!--<Setter Property="Width" Value="100" />-->
+        <!--<Setter Property="Height" Value="25" />-->
+        <!--<Setter Property="Clip" Value="" />-->
+        <!--<Setter Property="IsVisible" Value="True" />-->
+        <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource PrimaryLightColor}, Dark={StaticResource PrimaryDarkColor}}" />
```

Comment:
I wonder if we need to set background color here, or if Page is enough of a 'root' object to take care of any layouts?

Reply:
Should we not set a background color on any pages then and just let the system handle that? 

My thinking was that the background should be light in light mode, dark in dark mode, and since `BackgroundColor` property is derived from VisualElement, which Page is based on, this is how I handled it. Most of the specific pages themselves (i.e. ContentPage) don't have stylable properties if not for those derived from Page and VisualElement